scripts/update-image.py

```diff
@@ -895,7 +901,7 @@ def run(self):
     "-m",
     "--mount-using",
     dest="mount_using",
-    choices=["loopback", "docker", "guestfs", "block", "remake"],
+    choices=["loopback", "docker", "guestfs", "block"],
```

Comment:
where did remake go?

Reply:
remake is an action, not a way to mount the image (trying to do `-m remake` fails in the mount action) :^)